service/service.go

```diff
@@ -127,6 +128,12 @@ func New(ctx context.Context, set Settings, cfg Config) (*Service, error) {
 
 	logsAboutMeterProvider(logger, cfg.Telemetry.Metrics, mp, extendedConfig)
 	srv.telemetrySettings = component.TelemetrySettings{
+		LeveledMeterProvider: func(level configtelemetry.Level) metric.MeterProvider {
```

Comment:
Will the tests for these come later?

Reply:
yeah, when i move the code that uses the MeterProvider today to use the LeveledMeterProvider instead